sql_metadb/derived_tables/invoice_lines_adjustments.sql

```diff
@@ -0,0 +1,53 @@
+--invoice_lines_adjustments
+
+-- This table includes the extracted adjustments data on the invoice
+-- line level.  The field description can be locally defined by 
+-- institutions.  Examples are "shipping", "VAT" (MwSt), "Service Charge".
+-- The adjustment fund distribution data array will be extracted from the invoice_lines table
+-- in a separate derived table called invoice_lines_adjustments_fund_distributions in a future release.
+
+DROP TABLE IF EXISTS invoice_lines_adjustments;
+
+CREATE TABLE invoice_lines_adjustments AS
+WITH adjustments AS (
+    SELECT
+        id AS invoice_line_id,
+        jsonb_extract_path_text(adjustments.data, 'description') AS adjustment_description,
+--jsonb_extract_path_text(adjustments.data, 'fundDistributions') AS adjustment_fund_distributions,
+        jsonb_extract_path_text(adjustments.data, 'prorate') AS adjustment_prorate,
+        jsonb_extract_path_text(adjustments.data, 'relationToTotal') AS adjustment_relationToTotal,
```

Comment:
Uppercase letters should not be used in table or column name.  Please see the contributing guidelines.  I will make the corrections here.